### PR TITLE
Added quick links to all doc

### PR DESCRIPTION
**Pull request by TomLewis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +11,35 @@ Full featured clan system for PVP Minecraft Servers!
 [![Crowdin](https://badges.crowdin.net/simpleclans/localized.svg)](https://crowdin.com/project/simpleclans)
 [![Discord](https://img.shields.io/discord/719557355917934613?label=discord&logo=discord)](https://discord.gg/CkNwgdE)
 
-Spigot resource page: https://www.spigotmc.org/resources/simpleclans.71242/
+#### Download  
+
+* [Spigot Resource](https://www.spigotmc.org/resources/simpleclans.71242/)
+* [GitHub Releases](https://github.com/RoinujNosde/SimpleClans/releases)
```
Comment (RoinujNosde) [inline README.md]: This does not contain download links for newer releases.

**Review by RoinujNosde — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TomLewis) [reply README.md]: So you want it to link to https://github.com/RoinujNosde/SimpleClans/releases/latest ?

Comment (RoinujNosde) [reply README.md]: No need to link to GitHub releases. Releases contains only changelogs. The downloads are on Spigot (and Bintray, for devs).

Comment (TomLewis) [reply README.md]: oh right! ok now I understand. Let me remove it.

Comment (TomLewis) [reply README.md]: Ive updated it on my fork, I have no idea what I need to do to update it here

File: README.md [hunk 2]
```diff
@@ -44,5 +71,3 @@ dependencies {
     compileOnly "net.sacredlabyrinth.phaed.simpleclans:SimpleClans:2.12.2"
 }
 ```
-
-Some examples on how to use the API: https://github.com/RoinujNosde/SimpleClans/wiki/SimpleClans-API-example
```
Comment (RoinujNosde) [inline README.md:52]: I think it's best to leave this comment, as it's relevant to the Developers section.
(only the link needs fixing)

**Review by RoinujNosde — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (TomLewis) [reply README.md:52]: The link was moved to the top with the other navigation.

Comment (RoinujNosde) [reply README.md:52]: I saw that, but having it close to the maven stuff will be of more help as it's usually the first thing I look for when checking another repo.

Comment (TomLewis) [reply README.md:52]: Re-added with title.

Comment (RoinujNosde) [reply README.md:52]: Thanks

File: wiki/commands-and-permissions/permissions.md [hunk 3]
```diff
@@ -4,7 +4,9 @@ description: null
 
 # Permissions
 
-## SuperPerms Nodes
+## SuperPerms Nodes (Auto added)
```
Comment (RoinujNosde) [inline wiki/commands-and-permissions/permissions.md]: Maybe specify that not all super nodes are given.
"(Some are auto added)"

And on the list of super nodes put something like:
simpleclans.anyone.* **(auto added)**

**Review by RoinujNosde — CHANGES_REQUESTED** [review]

File: wiki/commands-and-permissions/permissions.md (hunk 3 above)
Comment (TomLewis) [reply wiki/commands-and-permissions/permissions.md]: Done! And re-wrote the description.

File: README.md [hunk 4]
```diff
@@ -9,9 +11,34 @@ Full featured clan system for PVP Minecraft Servers!
...
+  * [Clan Alliances and Rivalries](https://simpleclans.gitbook.io/simpleclans/commands-and-permissions/aliances-and-rivalries)
+  * [Clan Ranks with Permissions](https://simpleclans.gitbook.io/simpleclans/commands-and-permissions/ranks-with-permissions)
+* How to Setup
+  * [Configuration](https://simpleclans.gitbook.io/simpleclans/how-to-setup/configuration)
+  * [Translation](https://simpleclans.gitbook.io/simpleclans/how-to-setup/translation)
+  * [Clan Member Fee](https://simpleclans.gitbook.io/simpleclans/how-to-setup/member-fee)
+  * [Clan Upkeep](https://simpleclans.gitbook.io/simpleclans/how-to-setup/clan-upkeep)
+  * [Clan name above players head](https://simpleclans.gitbook.io/simpleclans/how-to-setup/clan-below-players-name)
+  * [Clan name on Tablist](https://simpleclans.gitbook.io/simpleclans/how-to-setup/clan-on-tablist)
+* Developers
```
Comment (RoinujNosde) [inline README.md]: Just noticed this.
PAPI and Land plugins are not for devs.
This title could be "Other".

**Review by RoinujNosde — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (TomLewis) [reply README.md]: I changed it to "plugins & development" as other isn't very descriptive.

Comment (RoinujNosde) [reply README.md]: Awesome

**Review by RoinujNosde — APPROVED** [review]

**Pull request merged**